protocol/app/testdata/default_genesis_state.json

```diff
@@ -350,8 +350,8 @@
     "params": {
       "treasury_account":"rewards_treasury",
       "denom":"dv4tnt",
-      "denom_exponent":-6,
-      "market_id":1,
+      "denom_exponent":-18,
+      "market_id":11,
```

Comment:
Whats the `market_id` for here? Should it be the same as the `MarketId` [here](https://github.com/dydxprotocol/v4-chain/blob/c491adc0054f61a94612c65a294c85b6c91cf57f/protocol/testutil/daemons/pricefeed/exchange_config/market_id.go#L82)?

Reply:
oh nvm saw thread below